performance_manager/lib/postgres_schema.py

```diff
@@ -6,63 +6,87 @@
 SqlBase: Any = declarative_base()
 
 
-class VehiclePositionEvents(SqlBase):  # pylint: disable=too-few-public-methods
-    """Table for GTFS-RT Vehicle Position Events"""
+class VehicleEvents(SqlBase):  # pylint: disable=too-few-public-methods
+    """
+    Table that hold GTFS-RT Events
 
-    __tablename__ = "eventsVehiclePositions"
+    Store all of the information that identifies a trip and a stop. Then tie that
+    information with timestamp events when a vehicle begins moring to the stop and
+    when it arrived at the stop ()
+    """
+
+    __tablename__ = "vehicle_events"
 
     pk_id = sa.Column(sa.Integer, primary_key=True)
-    is_moving = sa.Column(sa.Boolean)
-    stop_sequence = sa.Column(sa.SmallInteger, nullable=True)
-    stop_id = sa.Column(sa.String(60), nullable=True)
-    timestamp_start = sa.Column(sa.Integer, nullable=False)
-    timestamp_end = sa.Column(sa.Integer, nullable=False)
+
+    # trip identifiers
     direction_id = sa.Column(sa.SmallInteger, nullable=True)
```

Comment:
direction_id can be a bool type, now that we're planning on never having Null values in these fields